public/apps/login/login-page.tsx

```diff
@@ -237,7 +237,10 @@ export function LoginPage(props: LoginPageDeps) {
             );
           }
 
-          if (authOpts.length > 1) {
+          if (
+            authOpts.length > 1 &&
+            !(authOpts.includes(AuthType.PROXY) && authOpts.length === 2)
```

Comment:
why === 2 here?

Reply:
Because we need to know if proxy is used alongside another auth type. See Craig's original comment for a more in depth explanation. But in short, without this, the login page when proxy is with another type will get an ugly dividing bar that separates blank space.